homeassistant/components/media_player/sonos.py

```diff
@@ -42,6 +42,8 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     players = None
     hosts = config.get('hosts', None)
+    group_devices = config.get('group_devices', False)
```

Comment:
only interacting on groups / coordinators would be a good idea in my opinion but again, I don't know about all the complexities with other sonos devices and if there's a scenario but for example `socos.utils.requires_coordinator` actively replaces `player` with `coordinator` without any safety checks so it seems there's always a group and coordinator.

Let me know what you think and I'll update the PR :)